Dockerfile

```diff
@@ -1,16 +1,7 @@
 FROM agapito/centos7-scala:latest
 
-WORKDIR /root
+COPY . /twitter-lm-extractor/
 
-# install deps and create testdir
-RUN yum -y install make && \
-    mkdir -p /root/twitter-lm-extractor
+WORKDIR /twitter-lm-extractor
 
-WORKDIR /root/twitter-lm-extractor
-
-COPY build.sbt /root/twitter-lm-extractor
-COPY src /root/twitter-lm-extractor/src
-COPY project /root/twitter-lm-extractor/project
-
-# run tests
-CMD ["sbt", "clean", "test"]
+ENTRYPOINT ["sbt", "clean", "test"]
```

Comment:
You should change this to 

`ENTRYPOINT ["sbt", "-Dsbt.log.noformat=true", "clean", "test"]`

in order to stop sbt from printing out colour codes, which are not supported by the build server.